.github/workflows/pr-closed.yml

```diff
@@ -18,3 +18,4 @@ jobs:
       oc_token: ${{ secrets.OC_TOKEN }}
     with:
       cleanup: helm
+      packages: backend frontend migrations
```

Comment:
do we need this, since it is a HELM cleanup

Reply:
Yup!  We're promoting images to latest so builds don't have to keep happening.